src/main/interface/msp.c

```diff
@@ -1178,16 +1178,32 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU16(dst, gyroConfig()->gyro_soft_notch_hz_2);
         sbufWriteU16(dst, gyroConfig()->gyro_soft_notch_cutoff_2);
         sbufWriteU8(dst, currentPidProfile->dterm_filter_type);
+        sbufWriteU8(dst, currentPidProfile->dterm_filter_style);
         break;
+
 #ifdef USE_GYRO_FAST_KALMAN
-    case MSP_ADVANCED_FILTER_CONFIG :
-        sbufWriteU16(dst, gyroConfig()->gyro_soft_lpf_hz_2);
+    case MSP_FAST_KALMAN :
         sbufWriteU16(dst, gyroConfig()->gyro_filter_q);
         sbufWriteU16(dst, gyroConfig()->gyro_filter_r);
-        sbufWriteU16(dst, gyroConfig()->gyro_filter_p);
-        sbufWriteU8(dst, gyroConfig()->gyro_stage2_filter_type);
         break;
 #endif
+
+#ifdef USE_GYRO_IMUF9001
```

Comment:
Something whack going on with the TABS on this area - could we please get that harmonized to line up with the rest?

Reply:
done